src/Main.java

```diff
@@ -1,5 +1,116 @@
-        System.out.println("Hello world!");
+
+        System.out.println("Задача 1");
+        byte cat = 3;
+        short dog = 129;
+        int tree = 999999;
+        long box = 235699999999999L;
+        float sugar = 3.5F;
+        double water = 3.8976;
+
+        System.out.println("Значение переменной cat с типом byte равно " + cat);
+        System.out.println("Значение переменной dog с типом short равно " + dog);
+        System.out.println("Значение переменной tree с типом int равно " + tree);
+        System.out.println("Значение переменной box с типом long равно " + box);
+        System.out.println("Значение переменной sugar с типом float равно " + sugar);
+        System.out.println("Значение переменной water с типом double равно " + water);
+        System.out.println(" ");
+
+
+        System.out.println("Задача 2");
+        System.out.println("");
+
+        float donkey = 27.12f;
+        long bear = 987678965549L;
+        float velo = 2.786f;
+        short grass = 569;
+        short chocolate = -159;
+        short milk = 27897;
+        byte monkey = 67;
+
+        System.out.println("Задача 3");
+        short totalLists = 480;
+        byte class1 = 23;
+        byte class2 = 27;
+        byte class3 = 30;
+        byte totalPupils = (byte) (class1 + class2 + class3);
+        byte listsPerPupil = (byte) (totalLists / totalPupils);
+        System.out.println("На каждого ученика рассчитано " + listsPerPupil + " листов бумаги");
+        System.out.println("");
+
+        System.out.println("Задача 4");
+        byte capacityPerMinute = 8;
+
+        short minutesInDay = 60 * 24;
+        short capacityIn20Min = (short) (capacityPerMinute * 20);
+        short capacityInDay = (short) (minutesInDay * capacityPerMinute);
+        int capacityIn3Days = (int) capacityInDay * 3;
+        int capacityInMonth = (int) capacityInDay * 30;
+
+        System.out.println("За 20 минут машина произвела " + capacityIn20Min + " штук бутылок");
+        System.out.println("За сутки машина произвела " + capacityInDay + " штук бутылок");
+        System.out.println("За 3 суток машина произвела " + capacityIn3Days + " штук бутылок");
+        System.out.println("За месяц машина произвела " + capacityInMonth + " штук бутылок");
+        System.out.println("");
+
+        System.out.println("Задача 5");
+        byte whiteColourPerClass = 2;
+        byte brownColourPerClass = 4;
+        byte totalColours = 120;
+        byte classes = (byte) (totalColours / (whiteColourPerClass + brownColourPerClass));
+        byte totalWhiteColours = (byte) (whiteColourPerClass * classes);
+        byte totalBrownColours = (byte) (brownColourPerClass * classes);
+
+        System.out.println("В школе, где " + classes + " классов, нужно " + totalWhiteColours + " банок белой краски и " + totalBrownColours + " банок коричневой краски");
+        System.out.println("");
+
+        System.out.println("Задача 6");
+        short BananasGramm = 5 * 80;
```

Comment:
Имена переменных строго в camelCase, начиная с буквы в нижнем регистре
```suggestion
        short bananasGramm = 5 * 80;
```